go.mod

```diff
@@ -8,14 +8,15 @@ require (
 	github.com/gruntwork-io/terratest v0.43.12
 	github.com/hashicorp/go-cty v1.4.1-0.20200414143053-d3edf31b6320
 	github.com/hashicorp/hcl/v2 v2.17.0
+	github.com/hashicorp/terraform-plugin-log v0.9.0
 	github.com/hashicorp/terraform-plugin-sdk/v2 v2.27.0
 	github.com/hashicorp/terraform-plugin-testing v1.4.0
 	github.com/mongodb-forks/digest v1.0.5
 	github.com/mwielbut/pointy v1.1.0
 	github.com/spf13/cast v1.5.1
 	github.com/zclconf/go-cty v1.13.2
 	go.mongodb.org/atlas v0.33.0
-	go.mongodb.org/atlas-sdk/v20230201002 v20230201002.0.0
+	go.mongodb.org/atlas-sdk/v20230201005 v20230201005.0.1-0.20230825095504-8d6f690cf806
```

Comment:
Make sure to add the latest release https://github.com/mongodb/atlas-sdk-go/releases/tag/v20230201006.0.0 before merging the PR

Reply:
updated